<launch>
<node pkg="forklift_server" type="Topology_map_server.py" name="TopologyMap_server" output="screen">
    <param name="odom" value="/odom" /><!--里程計 Topic（map -> base_link, 是tf相對關係, 非輪式里程計）/-->
    <param name="start_node" value="A1" />
    <rosparam param= "graph">
    {
        "A_1":{"A_VS":1,"B_1":5},
        "A_VS":{"A_1":1,"A_2":1},
        "A_2":{"A_VS":1},
        "B_1":{"B_VS":1},
        "B_VS":{"A_2":5,"B_1":1}
    }
    </rosparam>
    <rosparam param= "waypoints">
    {
        "A_1":[4.887,   -7.760, 1.000,  0.000],
        "A_VS":[4.887,  -7.760, 0.707,  0.707],
        "A_2":[4.887,   -7.760, 0.000,  1.000],
        "B_1":[2.547,   -8.116, 1.000,  0.000],
        "B_VS":[2.547,  -8.116, 0.000,  1.000]
    }
    </rosparam>
</node>
</launch>
